frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -180,7 +180,7 @@ const ManageSchedulePage = ({
   const {
     data: globalScheduledQueries,
     error: globalScheduledQueriesError,
-    isFetching: isFetchingGlobalScheduledQueries,
+    isLoading: isLoadingGlobalScheduledQueries,
```

Comment:
This _might_ cause issues in cases where there are quick changes between page tabs. In react-query, `isLoading` represents the initial case where the query cache is entirely empty. If there is a cached value (which would typically be the case if the page has mounted at least once), react-query will return the cached value on the next page mount while it is doing a refetch in the background. So even if we are calling the fetch method directly, react-query itself will be utilizing the `isFetching` state. I think in most cases we found that react-query's `isFetching` aligns with the intended UX more so than its `isLoading`. It's not the most intuitive naming choice since it relates more to the state of the abstracted query cache rather than how developers tend to think about API state.  We probably want to consider how to best document patterns for react-query.